### PR TITLE
Mention CVE-2020-7021 under 7.10.0 release notes

### PR DESCRIPTION
**Pull request by pugnascotia**
Mention CVE-2020-7021 under 7.10.0 release notes

### CODE REVIEW

File: docs/reference/release-notes/7.10.asciidoc [hunk 1]
```diff
@@ -156,6 +156,19 @@ Snapshot/Restore::
 
 Also see <<breaking-changes-7.10,Breaking changes in 7.10>>.
 
+[discrete]
+[[security-updates-7.10.0]]
+=== Security updates
+
+* Elasticsearch versions before 7.10.0 and 6.8.14 have an information
```
Comment (jrodewig) [inline docs/reference/release-notes/7.10.asciidoc:163]:
```suggestion
* {es} versions before 7.10.0 and 6.8.14 have an information
```

File: docs/reference/release-notes/7.10.asciidoc [hunk 2]
```diff
@@ -156,6 +156,19 @@ Snapshot/Restore::
...
+disclosure issue when audit logging and the emit_request_body option is
```
Comment (jrodewig) [inline docs/reference/release-notes/7.10.asciidoc:164]:
```suggestion
disclosure issue when audit logging and the `emit_request_body` option are
```

File: docs/reference/release-notes/7.10.asciidoc [hunk 3]
```diff
@@ -156,6 +156,19 @@ Snapshot/Restore::
...
+enabled. The {es} audit log could contain sensitive information
```
Comment (jrodewig) [inline docs/reference/release-notes/7.10.asciidoc:165]:
```suggestion
enabled. The {es} audit log could contain sensitive information,
```

File: docs/reference/release-notes/7.10.asciidoc [hunk 4]
```diff
@@ -156,6 +156,19 @@ Snapshot/Restore::
...
+such as password hashes or authentication tokens. This could allow an
+{es} administrator to view these details.
+You must upgrade to {es} version 7.10.2 to obtain the fix.
+https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7021
```
Comment (jrodewig) [inline docs/reference/release-notes/7.10.asciidoc:169]:
```suggestion
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7021[CVE-2020-7021]
```

**Review by jrodewig — APPROVED** [review]
👍  LGTM. I left a few minor nits but nothing blocking. Thanks @pugnascotia.

File: docs/reference/release-notes/7.10.asciidoc [hunk 5]
```diff
@@ -160,13 +160,13 @@ Also see <<breaking-changes-7.10,Breaking changes in 7.10>>.
...
 
-* Elasticsearch versions before 7.10.0 and 6.8.14 have an information
-disclosure issue when audit logging and the emit_request_body option is
-enabled. The {es} audit log could contain sensitive information
+* {es} versions before 7.10.0 and 6.8.14 have an information
+disclosure issue when audit logging and the `emit_request_body` option are
+enabled. The {es} audit log could contain sensitive information,
 such as password hashes or authentication tokens. This could allow an
 {es} administrator to view these details.
 You must upgrade to {es} version 7.10.2 to obtain the fix.
```
Comment (pugnascotia) [inline docs/reference/release-notes/7.10.asciidoc:168]: @jrodewig I missed this - this should be 7.10.0, yes?

Comment (jrodewig) [reply docs/reference/release-notes/7.10.asciidoc:168]: :doh: Sorry about that I overlooked this too. Yes. This should be 7.10.0.

Comment (jrodewig) [reply docs/reference/release-notes/7.10.asciidoc:168]: However, I'm not sure it's worth updating... It should be impossible for users to obtain an earlier patch version now.

Comment (pugnascotia) [reply docs/reference/release-notes/7.10.asciidoc:168]: I pushed a fixed anyway.

**Comment by pugnascotia** [comment]
Backported to `6.8` in facdf64c1f5

**Comment by Sachpat** [comment]
HI @pugnascotia , is it also backported to 7.9.x ? Or 7.9.x is still vulnerable?

**Comment by pugnascotia** [comment]
@Sachpat You must upgrade to Elasticsearch v7.10.2 to obtain the fix.

**Comment by pugnascotia** [comment]
You can read more here: https://discuss.elastic.co/t/elastic-stack-7-11-0-and-6-8-14-security-update/263915

**Comment by pugnascotia** [comment]
Forward-ported to `7.11` and `7.x`

**Pull request merged**